docs/modules/customize/pages/cookies.adoc

```diff
@@ -0,0 +1,80 @@
+= Cookies
+
+With cookie consent we can explain better what cookies do we use, why we're using these cookies, and let the user choose which cookies they want to use.
+
+== Different cookie categories available in Decidim
+
+By default there are four (4) cookie categories: **essential**, **preferences**, **analytics** and **marketing**. To display embedded iframes properly user has to give consent to all categories.
+
+== Adding scripts which require cookie consent
+
+Recommended place for scripts is ```app/views/layouts/decidim/_head_extra.html.erb```.
+For example if you want to add script that requires marketing consent, you can add following code:
+
+```
+<script type="text/plain" data-consent="marketing">
+  console.log('marketing cookies accepted');
+</script>
+```
+
+**Note that script type should be "text/plain"!** Without that user's cookie consent setting doesn't affect.
+
+== Adding cookie category and cookie documentation
+
+You are able to modify cookie categories via consent_gategories configuration option.
+
+For example if you want to add "statistics" category you need to add following code to your app's initializer:
+
+```
+Decidim.configure do |config|
+  config.consent_categories = [
+    {
+      slug: "essential",
+      mandatory: true,
+      cookies: [
+        {
+          type: "cookie",
+          name: Decidim.consent_cookie_name
+        }
+      ]
+    },
+    {
+      slug: "preferences",
+      mandatory: false
+    },
+    {
+      slug: "analytics",
+      mandatory: false
+    },
+    {
+      slug: "marketing",
+      mandatory: false
+    },
+    {
+      slug: "statistics",
+      mandatory: false,
+      cookies: [
+        type: "cookie",
+        name: "statistics-cookie"
+      ]
+    }
+  ]
+end
+
+```
+
+When adding cookie categories and cookies also remember to add following translations:
+
+For category:
+```
+layouts.decidim.cookie_consent.modal.statistics.title
+layouts.decidim.cookie_consent.modal.statistics.description
+```
+
+For cookie:
+```
+layouts.decidim.cookie_consent.cookie_details.cookies.statistics-cookie.description
+layouts.decidim.cookie_consent.cookie_details.cookies.statistics-cookie.service
+```
+* Description explains the purpose of using this cookie
+* Service explains the 3rd party service where this cookie is coming from
```

Comment:
```suggestion
* Service explains the 3rd party service where this cookie is coming from

So, for instance for the English language (`en`), it would look like this:

```yaml
en:
  layouts:
    decidim:
      cookie_consent:
        modal:
          statistics:
            title: The title for the statistics
            description: The description for the statistics
        cookie_details:
          cookies:
            statistics-cookie:
              description: The purpose of the cookie
              service: The 3rd party service of the cookie
```
```


I like to leave the documentation "copy paste" friendly.